webviews/editorWebview/index.css

```diff
@@ -322,7 +342,7 @@ body .comment-container .comment-body > div > p:last-child,
 body {
 	margin: auto;
 	width: 100%;
-	max-width: 925px;
+	max-width: 1280px;
```

Comment:
This happens on my 12 inch Surface Pro:

![image](https://user-images.githubusercontent.com/46387399/195891572-36cee87a-e26d-4e81-b1c8-99b824bfd4d5.png)